docs/api/index.md

```diff
@@ -1,39 +1,129 @@
 ---
-sidebar_position: 1
-title: Introduction
+sidebar\_position: 1
```

Comment:
Is the `\` here intentional or accidental?

Reply:
my editor tells me that this is the right thing to do when editing a markdown doc. i think it's confused because it's the yaml front matter but in a markdown document you need to escape an underscore if you don't want to use it for _emphasis_

Reply:
Ah. Let's remove them in the front-matter to be consistent with the rest of the codebase